other/housing/housing.jl

```diff
@@ -1,12 +1,32 @@
+# # Machine Learning Problem : Housing Dataset
+#
+# The housing problem functions as a starting point in Machine Learning.
+# We'll be demonstrating the use of Julia's [Flux Package](https://fluxml.ai/)
+# to do this problem.
+#
+# The data replicates the housing data example from the Knet.jl readme. Although we
+# could have reused more of Flux (see the mnist example), the library's
+# abstractions are very lightweight and don't force you into any particular
+# strategy.
+# 
+# [This](http://www.mit.edu/~6.s085/notes/lecture3.pdf) might help you know more about the fundamentals of what 
+# we're about to do. If you don't understand something there which is also not mentioned here in this file, 
+# you may overlook that (or search it up on google to quench your curiosity :-)
+
 using Flux.Tracker, Statistics, DelimitedFiles
 using Flux.Tracker: Params, gradient, update!
 using DelimitedFiles, Statistics
 using Flux: gpu
 
-# This replicates the housing data example from the Knet.jl readme. Although we
-# could have reused more of Flux (see the mnist example), the library's
-# abstractions are very lightweight and don't force you into any particular
-# strategy.
```

Comment:
While defining the problem is indeed important, it is equally important to demonstrate the pattern of programming that we are trying to showcase here. Removing this means there is no good way of pinpointing why it belongs in the model zoo.

Reply:
I've not removed that, just shifted it up. 😃 
See lines 7-10.